### PR TITLE
hate speech 18 dataset

### PR DESCRIPTION
**Pull request by czabo**
This is again a PR instead of #1339, because something went wrong there.

### CODE REVIEW

File: datasets/hate_speech18/README.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+      Cuadros, Montse",
+    booktitle = "Proceedings of the 2nd Workshop on Abusive Language Online ({ALW}2)",
+    month = oct,
+    year = "2018",
+    address = "Brussels, Belgium",
+    publisher = "Association for Computational Linguistics",
+    url = "https://www.aclweb.org/anthology/W18-5102",
+    doi = "10.18653/v1/W18-5102",
+    pages = "11--20",
+}
```
Comment (yjernite) [inline datasets/hate_speech18/README.md:156]: Could you put this in a code black?

Comment (czabo) [reply datasets/hate_speech18/README.md]: Done, thanks!

File: datasets/hate_speech18/README.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+multilinguality:
+- monolingual
+size_categories:
+- 10K<n<100K
+source_datasets:
+- original
+task_categories:
+- text-classification
+task_ids:
+- intent-classification
```
Comment (lhoestq) [inline datasets/hate_speech18/README.md:18]:
```suggestion
---
annotations_creators:
- found
language_creators:
- found
languages:
- en
licenses:
- cc-by-sa-3.0
multilinguality:
- monolingual
size_categories:
- 10K<n<100K
source_datasets:
- original
task_categories:
- text-classification
task_ids:
- intent-classification
---
```

**Review by lhoestq — APPROVED** [review]
Thanks ! LGTM :)

**Comment by lhoestq** [comment]
The error `tests/test_file_utils.py::TempSeedTest::test_tensorflow` just appeared because of tensorflow's update.
Once it's fixed on master we'll be free to merge this one

**Comment by lhoestq** [comment]
It's fixed on master now :) 

merging this once

**Pull request merged**